recipe/meta.yaml

```
@@ -26,10 +28,13 @@ requirements:
 
 test:
   commands:
-    - test -f ${PREFIX}/include/libftdi1/ftdi.h  # [unix]
-    - test -f ${PREFIX}/lib/libftdi1${SHLIB_EXT}  # [unix]
-    - test -f ${PREFIX}/lib/pkgconfig/libftdi1.pc  # [unix]
+    - test -f ${PREFIX}/include/libftdi1/ftdi.h                  # [unix]
+    - test -f ${PREFIX}/lib/libftdi1${SHLIB_EXT}                 # [unix]
+    - test -f ${PREFIX}/lib/pkgconfig/libftdi1.pc                # [unix]
     - test -f ${PREFIX}/lib/cmake/libftdi1/LibFTDI1Config.cmake  # [unix]
+    - if not exist %LIBRARY_INC%\libftdi1\ftdi.h (exit 1)        # [win]
+    # - if not exist %LIBRARY_LIB%\libftdi1.lib (exit 1)           # [win]
+    - if not exist %LIBRARY_BIN%\libftdi1.dll (exit 1)           # [win]
```

```suggestion
    - if not exist %LIBRARY_LIB%\ftdi1.lib (exit 1)           # [win]
    - if not exist %LIBRARY_BIN%\ftdi1.dll (exit 1)           # [win]
```
Windows doesn't put 'lib' in front of libraries by default, so this is what they will be called.